File: 25. Find the fine/main.cpp
// User function Template for C++
class Solution 
{
  public:
    long long int totalFine(int date, vector<int> &car, vector<int> &fine) 
    {
        // code here
        long long int total_fine = 0;
        long long int n = car.size();
        
        for (long long int i = 0; i < n; i++)
            if ((car[i] + date) & 1)
                total_fine += fine[i];
        
        return total_fine;
    }
};

File: 25. Find the fine/main.py
#User function Template for python3
class Solution:
    def totalFine(self, date, car, fine):
        #Code here
        total_fine = 0
        
        for i in range(len(car)):
            if (car[i] + date) & 1:
                total_fine += fine[i]
        
        return total_fine